python/chronos/dev/test/run-pytests.sh

```diff
@@ -60,7 +60,7 @@ python -m pytest -v test/bigdl/chronos/autots\
                     test/bigdl/chronos/data \
                     test/bigdl/chronos/simulator \
                     test/bigdl/chronos/detector \
-       -k "not test_forecast_tcmf_distributed"
+       -k "not deprecated"
```

Comment:
only `not deprecated` is enough?

Reply:
I'm sure other tf1 ut will be skipped, so we just need to specify "-k not deprecated".